src/app/components/forms/game/sketchfab/sketchfab.ts

```diff
@@ -1,17 +1,56 @@
-import { BaseForm, FormOnInit } from '../../../../../_common/form-vue/form.service';
+import { Component, Prop } from 'vue-property-decorator';
+import { BaseForm, FormOnInit, FormOnSubmit } from '../../../../../_common/form-vue/form.service';
 import { Game } from '../../../../../_common/game/game.model';
 import { GameSketchfab } from '../../../../../_common/game/sketchfab/sketchfab.model';
-import { Component, Prop } from 'vue-property-decorator';
+import AppSketchfabEmbed from '../../../../../_common/sketchfab/embed/embed.vue';
 
-@Component({})
-export default class FormGameSketchfab extends BaseForm<GameSketchfab> implements FormOnInit {
+@Component({
+	components: {
+		AppSketchfabEmbed,
+	},
+})
+export default class FormGameSketchfab extends BaseForm<GameSketchfab>
+	implements FormOnInit, FormOnSubmit {
 	@Prop(Game) game!: Game;
 
+	readonly SKETCHFAB_URL_REGEX_1 = /^(?:(?:https:\/\/)?(?:www.)?(?:sketchfab.com\/3d-models\/(?:[a-z0-9]+-)+([0-9a-f]{32})\/?))$/i;
```

Comment:
Are you sure the first part after the `3d-models` bit isn't part of the hash that would identify the model?
e.g. `artist-abcde124` instead of just `abcde124`

Reply:
also the `.` in `www.` should be escaped to be `www\.` Same with the `.` in `sketchfab.com`

Reply:
The new Sketchfab urls look like this:
https://sketchfab.com/3d-models/lowpoly-80s-tv-70001ecdc29c43ceba1d766f09fe79c0
which have the name slugified and then the id of the model.